### DISCUSSION TITLE
可用于《宝可梦》第四世代汉化修正版的PKHeX版本

### DISCUSSION THREAD

**Discussion by giscus-xzonn** [Xzonn/PokemonChineseTranslationRevise#22] [General]
# 可用于《宝可梦》第四世代汉化修正版的PKHeX版本

https://blog.www.localhost/PokemonChineseTranslationRevise/PKHeX.html

**Comment by Xzonn** [comment]
> 评论导出自来必力

*****b***h 评论于 2020-04-29 01:47:36 (UTC+8)*****

请问下您有没有兴趣关注一下登入名人堂（日文︰殿堂入り，英文︰entering the Hall of Fame）这个界面，它记录的是每一次通关时使用的宝可梦，可以在PC中心的电脑上看到。  
如果使用旧版汉化的存档，在你的官汉版上显示宝可梦的名字就会是乱码（应该是字库原因）。我自己尝试查找修改了一下，在白金里它第一次登入名人堂记录的第一只PM的位置应该在0x0001FFF2左右开始，HGSS和DPPt还没有尝试成功……  
感觉这样可以极大满足强迫症患者完美转移存档的想法……

![https://d47jbcq60tnr6.cloudfront.net/2020428/8332-3jso8h.eviy.png](https://d47jbcq60tnr6.cloudfront.net/2020428/8332-3jso8h.eviy.png)

*****b***h 回复于 2020-04-29 01:50:01 (UTC+8)*****

对了还是要万分感谢你的官汉版本，特别是DPPt这种从来没有官汉的版本，很多地名和人名我还是第一次知道他们的名字 :stuck_out_tongue_closed_eye:

**Reply by vast0618** [reply]
同求修复名人堂乱码问题 Orz

**Comment by Xzonn** [comment]
> 评论导出自来必力

*****眼睛是沉睡的湖 评论于 2020-08-14 12:35:17 (UTC+8)*****

打开时 报错  
PKHeX Error

A fatal non-UI error has occurred in PKHeX, and there was a problem displaying the details. Please report this to the author.  
怎么办呢

**Comment by Xzonn** [comment]
> 评论导出自来必力

*****卧看微尘 评论于 2021-03-24 11:56:37 (UTC+8)*****

大佬您好，想请问下pkhex源代码里，码表和宝可梦名称的文档是在那里？翻了老半天，没有找到具体是哪个文件

**Comment by Xzonn** [comment]
> 评论导出自来必力

*****圣空翔翼 评论于 2021-06-25 18:09:45 (UTC+8)*****

老哥你好，能帮帮强迫症吗，心金魂银换了这个rom以后劲敌名字乱码了，修改器好像没有改劲敌名字的功能

**Comment by Xzonn** [comment]
> 评论导出自来必力

*****丛逸东 评论于 2022-03-27 18:14:22 (UTC+8)*****

帅气的大佬 最近考古火红叶绿，有最早群星汉化能用的PKHex吗？好人一生平安

**Comment by Xzonn** [comment]
> 评论导出自来必力

*****朴***言 评论于 2022-11-26 23:22:16 (UTC+8)*****

能更新黑白2?
